capstone/src/main/java/com/google/sps/Objects/response/ChallengeResponse.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps.Objects.response;
+
+import com.google.sps.Objects.Challenge;
+
+/**
+ * This class is used in the ChallengeServlet to wrap together the current challenge and whether the
+ * current user has completed the challenge.
+ *
+ * @author lucyqu
+ */
+public class ChallengeResponse {
+
+  private Challenge challenge;
+  private boolean isCompleted;
```

Comment:
Please also change all of the isCompleteds here to hasUserCompleted

Reply:
I tried doing this and went back and forth several times but for whatever reason when I change it to hasUserCompleted, the checkbox status doesn't save when the page is reloaded. Not sure what the issue is